src/prif_m.f90

```diff
@@ -0,0 +1,11 @@
+! Copyright (c), The Regents of the University of California
+! Terms of use are as specified in LICENSE.txt
+module prif_m
```

Comment:
@everythingfunctional This PRIF module name is not currently part of the spec draft, but IIUC it probably should be.

How common is this `_m` suffix convention? Is `prif_m` what we should adopt?

Reply:
`_m` is a common enough convention, but I don't think it's needed in this context. I think the entire interface should be defined in a single module named `prif`. I'll add it to the design doc.

Reply:
Actually, it was already there.

> The procedures and types provided
for direct invocation as part of the runtime library shall be defined in a
Fortran module with the name `prif`.